pkg/agent/route/route_linux.go

```diff
@@ -1267,37 +1267,53 @@ func (c *Client) initNodeLatency() {
 	}
 
 	buildInputRule := func(ipProtocol iptables.Protocol, icmpType int32) string {
+		var comment string
+		if icmpType == 128 {
+			comment = "Antrea: allow ICMP echo request ingress packets from NodeLatencyMonitor"
+		} else if icmpType == 8 {
+			comment = "Antrea: allow ICMP echo ingress packets from NodeLatencyMonitor"
+		} else {
+			comment = "Antrea: allow ICMP echo reply ingress packets from NodeLatencyMonitor"
+		}
 		return iptables.NewRuleBuilder(antreaInputChain).
 			MatchInputInterface(gateway).
 			MatchICMP(&icmpType, nil, ipProtocol).
-			SetComment("Antrea: allow ICMP ingress packets from NodeLatencyMonitor").
+			SetComment(comment).
 			SetTarget(iptables.AcceptTarget).
 			Done().
 			GetRule()
 	}
 	buildOutputRule := func(ipProtocol iptables.Protocol, icmpType int32) string {
+		var comment string
+		if icmpType == 128 {
+			comment = "Antrea: allow ICMP echo request egress packets from NodeLatencyMonitor"
+		} else if icmpType == 8 {
```

Comment:
no need to distinguish between the 2, they are both for ICMP echo requests (one for IPv4 and one for IPv6)

personally i think we can use the same comment everywhere, for both request and reply, and avoid the complexity (e.g. `Antrea: allow ICMP probes for NodeLatencyMonitor`).